[submodule]Update fbgemm submodule pinned version

gchalump opened the pull request:
Differential Revision: D74335570

Skylion007 reviewed (approved):
NICE! We finally have FBGEMM updating and building after 2 years! <3

cyyever reviewed (approved):
Nice

cyyever commented:
You may need to wait until I have updated CMake to newer versions.

gchalump committed:
Update fbgemm pinned version (#153072)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/153072

Update fbgemm pinned version.
Related update in fbgemm: D74434751

Included changes:
 - Update fbgemm external dependencies directory in setup.py

Test Plan: PyTorch CI

Differential Revision: D74335570

yangw-dev commented:
@pytorchbot merge

yangw-dev commented:
@pytorchbot merge

yangw-dev commented:
close the pr since the user try to revert internally, and
[D75023416]
Revert D74335570
